src/raftstore/store/fsm/store.rs

```diff
@@ -1022,6 +1022,14 @@ impl RaftBatchSystem {
         self.apply_system
             .schedule_all(region_peers.iter().map(|pair| pair.1.get_peer()));
 
+        let router = Mutex::new(self.router.clone());
+        pd_client.handle_reconnect(move || {
+            router
+                .lock()
+                .unwrap()
+                .broadcast_normal(|| PeerMsg::HeartbeatPd);
```

Comment:
How about `PeerMsg::Tick(HeartbeatPd)`?

Reply:
Do you mean replace the message with PeerTicks? I think it's a good idea to separate automatically tick from custom messages.